scripts/optimize-assets.sh

```
@@ -0,0 +1,470 @@
+            log_warn "Already processed: ${basename}"
+            SKIPPED=$((SKIPPED + 1))
+            continue
+        fi
+
+        if [ "$DRY_RUN" = true ]; then
+            log_dry "Would convert: ${basename} -> ${sanitized}.ogg (${AUDIO_BITRATE})"
+            continue
+        fi
+
+        log_info "Converting audio: ${basename} -> ${sanitized}.ogg"
+        if ffmpeg -y -i "$src_file" -c:a libvorbis -b:a "$AUDIO_BITRATE" -ar 44100 "$out_file" -loglevel error 2>/dev/null; then
```

Shellcheck (0.11.0)

[warning] Use ffmpeg -nostdin to prevent ffmpeg from swallowing stdin.

(SC2095)